packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -269,7 +277,7 @@ const VirtualListBaseFactory = (type) => {
 		}
 
 		componentWillUpdate (nextProps, nextState) {
-			if (this.state.firstIndex === nextState.firstIndex) {
+			if (this.state.firstIndex === nextState.firstIndex || this.props.itemProps && this.props.itemProps !== nextProps.itemProps) {
```

Comment:
Umm. Do we need to consider the case that the prop is _undefined_ at first then assigned later for updating?

Reply:
No. I don't think so.